src/vm/gdbjit.cpp

```diff
@@ -2246,7 +2250,11 @@ bool NotifyGdb::BuildSymbolTableSection(MemBuf& buf, PCODE addr, TADDR codeSize)
         sym[i].st_other = 0;
         sym[i].st_shndx = SectionNamesCount + (i - (1 + method.GetCount())); // .thunks section index
         sym[i].st_size = 8;
+#ifdef _TARGET_ARM_
```

Comment:
Could you please use the `ThumbCodeToDataPointer<TADDR,PCODE>(...)` function to clear the THUMB bit?